src/Web/Static/documents/releases/weekly.html.cshtml

```diff
@@ -2,16 +2,35 @@
 layout: herosmall
 title: Weekly Releases
 image: /content/red_smoke_4.jpg
-releases?: query documents where sourceRelativePath startswith "documents\releases\v" descending date take 5
+releasesV3?: query documents where sourceRelativePath startswith "documents\releases\v3" descending date take 5
+releasesV4?: query documents where sourceRelativePath startswith "documents\releases\v4" descending date take 5
 
 ---
 
-
 <h2>Weekly Releases</h2>
-<p>The following weekly releases are available. These development releases may be unstable and are therefore not recommended for production use. They are purged irregularly. Please upgrade often (we recommend at least monthly).</p>
-<ul>
-@foreach (var release in @Model.Document.Releases)
-{
- <li><a href="@release.RelativeUrl">@Raw(release.Title)</a> on @release.Date.ToString("dddd, MMMM dd yyyy")</li>
-}
-</ul>
+<p>
+    The following weekly releases are available. These development releases may be unstable and
+    are therefore not recommended for production use. They are purged irregularly. Please upgrade 
+    often (we recommend at least monthly).
+</p>
+
+<div class="col-xs-5">
+    <h3>WiX v3.x Release Chain</h3>
```

Comment:
As both columns are subordinate to `Weekly Releases`, I'd say you could omit it entirely and just say `WiX v3.x` and `WiX v4.x`.